src/data/resolvers/motions.ts

```diff
@@ -158,6 +158,28 @@ export const motionsResolvers = ({
       const timeToSubmitMS = motion.events[1].toNumber() * 1000;
       const timeToSubmitInPast = timeToSubmitMS < blocktime;
 
+      const newestMotionStakedEvent = sortedEvents.find(
```

Comment:
why newest? do we have an earlier one? if yes maybe latest is a better prefix? newest doesn't sound grammatically correct. It's not a big deal though.

Reply:
They are synonyms, grammatically they both are valid. 

However, I think it's more common to say "latest event" than "newest event" for example, actually never heard anyone refer to a recent event as "newest" in english :)

So yeah, I could change it just for the sake of it sounding right, or more "native-like"